spec/controllers/api/v1/books_controller_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+require 'factories/book_factory'
+require 'factories/user_factory'
+require 'shared_contexts/authenticated_user.rb'
+
+RSpec.describe Api::V1::BooksController, type: :controller do
+  include_context 'Authenticated User'
+
+  describe 'GET #index' do
+    context 'When fetching all books' do
+      let!(:books) { create_list(:book, 3) }
+
+      before do
+        get :index, params: { id: user.id }
```

Comment:
There are project that use other convention, and try to define a `subject` for each `describe`